angular-ngrx-scss/src/app/state/user/user.effects.ts

```diff
@@ -15,7 +15,7 @@ export class UserEffects {
     return this.actions$.pipe(
       ofType(fetchUserData),
       switchMap(() =>
-        this.userService.getUserInfo().pipe(
+        this.userService.getAuthenticatedUserInfo().pipe(
```

Comment:
Potentially needs to be switched back if we don't keep both calls.

Reply:
We should discuss this later. I think I have some missing context.